test/utils/InstanceDefinitionUtils.test.ts

```diff
@@ -111,11 +120,29 @@ describe('InstanceDefinitionUtils', () => {
   });
 
   describe('Non-example instances', () => {
```

Comment:
This is definitely my bad, but since these tests are not linked to each other, I feel like we don't actually need this separate `describe`.  How do you feel about getting rid of that wrapper around the rest of these tests?

Reply:
Sounds good, I will change that!